README.adoc

```diff
@@ -385,6 +385,28 @@ apidocs:
 ...
 ---
 
+=== LDAP Configuration
+To configure authentication strategy to LDAP add the following configuration to the yaml file. If the auth strategy is LDAP. The application will not start without the basic LDAP settings like ldap_host, ldap_port, ldap_base and ldap_bind_dn.
```

Comment:
Not really a blocker but it may be helpful to indicate that the values below are just example values that should be changed.

Also,  omit "If the auth strategy is LDAP."

Reply:
I re-wrote this a bit.